test/runner/requirements/integration.cloud.hcloud.txt

```diff
@@ -1 +1 @@
-hcloud ; python_version >= '2.7' and python_version <= '3.7'  # Python 2.6 and 3.8 are not supported
+hcloud>=1.3.0 ; python_version >= '2.7' and python_version <= '3.7'  # Python 2.6 and 3.8 are not supported
```

Comment:
This will fix the CI failure. Can you update the comment to explain why version 1.3.0 or later is needed?

```suggestion
hcloud>=1.3.0 ; python_version >= '2.7' # Python 2.6 is not supported (sanity_ok)
```